lib/routes/extensions.js

```diff
@@ -12,3 +12,8 @@ function setupPathsFor (item) {
 setupPathsFor('scripts')
 setupPathsFor('stylesheets')
 setupPathsFor('assets')
+
+if (extensions.legacyGovukFrontendFixesNeeded()) {
+  requests.serveDirectory('/assets',
```

Comment:
Is there a reason we can't serve govuk-frontend assets from `/extension-assets/govuk-frontend/assets`?

Reply:
We will once the relevant changes have been made in `govuk-frontend`, they're relying on us having this line at the moment but the [PR I raised with them](https://github.com/alphagov/govuk-frontend/pull/2851) removes that assumption.  When that PR's merged they'll be using the `/extension-assets/govuk-frontend/assets`, but for now both the CSS and Nunjucks layout are expecting this URL.

Reply:
Oh, how come we can't set `$govuk-assets-path` the same way we set the global styles?

Reply:
We can, but that doesn't cover the URLs in the template.  We could do them separately by passing in parameters to Nunjucks ... I guess that would make it a more accurate representation of how things will be after Frontend moves to the standard approach.